File: Fase 2/revistas-app/src/app/objetos/base/Pago.ts
export class Pago {
    private _usuario: string;
    private _suscripcion: number;
    private _fechaPago: Date;
    private _periodo: string;
    private _tiempoPagado: number;
    
    constructor(usuario: string, suscripcion: number, fechaPago: Date, periodo: string, tiempoPagado: number) {
        this._usuario = usuario;
        this._suscripcion = suscripcion;
        this._fechaPago = fechaPago;
        this._periodo = periodo;
        this._tiempoPagado = tiempoPagado;
    }
    
    public get usuario(): string {
        return this._usuario;
    }
    public get suscripcion(): number {
        return this._suscripcion;
    }
    public get fechaPago(): Date {
        return this._fechaPago;
    }
    public get periodo(): string {
        return this._periodo;
    }
    public get tiempoPagado(): number {
        return this._tiempoPagado;
    }
}

File: Fase 2/ServidorRevistas/src/main/java/objetos/Pago.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package objetos;

import java.util.Date;

/**
 *
 * @author fernanrod
 */
public class Pago {
    private String usuario;
    private Integer suscripcion;
    private Date fechaPago;
    private String periodo;
    private Integer tiempoPagado;

    public Pago(String usuario, Integer suscripcion, Date fechaPago, String periodo, Integer tiempoPagado) {
        this.usuario = usuario;
        this.suscripcion = suscripcion;
        this.fechaPago = fechaPago;
        this.periodo = periodo;
        this.tiempoPagado = tiempoPagado;
    }
    
    public String getUsuario() {
        return usuario;
    }
    public Integer getSuscripcion() {
        return suscripcion;
    }
    public Date getFechaPago() {
        return fechaPago;
    }
    public String getPeriodo() {
        return periodo;
    }
    public Integer getTiempoPagado() {
        return tiempoPagado;
    }
}
